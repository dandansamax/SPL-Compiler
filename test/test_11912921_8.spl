int test_8()
{
    int var = 30;
    float f = 3.2;
    var(33) = f;
}