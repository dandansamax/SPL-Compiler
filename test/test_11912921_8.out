Error type 6 at Line 5: rvalue appears on the left-hand side of the assignment operator: 
Error type 11 at Line 5: invoking non-function variable: var
